mlflow/utils/databricks_utils.py

```diff
@@ -260,6 +260,19 @@ def get_job_type():
     except Exception:
         return _get_context_tag("jobTaskType")
 
+@_use_repl_context_if_available("jobType")
+def get_job_type_info():
+    try:
+        return _get_context_tag("jobType")
+    except Exception:
+        None
+
+def get_job_to_experiment_name_mapping():
```

Comment:
I think` def get_experiment_name_from_job_id(job_id)` is cleaner.

Reply:
+1